integr8ly/tutorial-web-app
# external walkthroughs

## Motivation

Run the walkthroughs from an external (to the webapp) location. This PR removes the walkthroughs from the webapp public directory and expects them to be present in either `../tutorial-web-app-walkthroughs` or a location passed in the `start:dev` script via `WALKTHROUGH_LOCATIONS` (you need to edit it in package.json since the cmdline arg will be overridden when running `yarn start:dev`).

## Verification steps:

1. Clone https://github.com/integr8ly/tutorial-web-app-walkthroughs to the same root directory as the webapp
1. Run `yarn start:dev`
1. Try out some walkthroughs. Everything should work.
